docs/source/protocol.rst

```diff
@@ -30,28 +30,29 @@ Network Send Number
 - Begins at 0 on each client and goes up by 1 each time we attempt a metrics network upload.
 - Does not go up on network retries.
 
-Relative Timestamp
-++++++++++++++++++
+Image
++++++
 
-- 64 bit signed integer
-- GVariant symbol: ``x``
-- See: http://linux.die.net/man/3/clock_gettime
+- String
+- GVariant symbol: ``s``
```

Comment:
We should copy the description & purpose of this string from the `ImageVersion` table and ideally document its structure (from `parse_endless_os_image()`).

Reply:
This is marked as resolved but it doesn't seem to be resolved.

Reply:
This was marked resolved but doesn't seem to be.

Reply:
The commit was on my computer, but not pushed… I’ll reapply the commit on top of the pushed-force new head.